demo/markdownlint-browser.js

```diff
@@ -4643,11 +4644,10 @@ module.exports = {
             let match = null;
             while ((match = htmlElementRe.exec(token.content)) !== null) {
                 const [tag, , element] = match;
-                if (element.toLowerCase() === "a") {
-                    const idMatch = identifierRe.exec(tag);
-                    if (idMatch) {
-                        fragments.set(`#${idMatch[1]}`, 0);
-                    }
+                const idMatch = identifierRe.exec(tag) ||
```

Comment:
maybe `anchorMatch` would be better since it's not just IDs

Reply:
It's always matching on "id", just not limiting to "a" tags.

Reply:
I think I get your point now, open to this.

Reply:
maybe `anchorMatched` is a little clearer

Reply:
Agreed. Done. Went with `anchorMatch` since this is a "match" object, not a "matched" boolean.